src/Insurance/App/Transformers/PayerTransformer.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Insurance\App\Transformers;
+
+class PayerTransformer
```

Comment:
is this empty?

Reply:
yep, for now I didn't add any data because the original version didn't include any data from the payer and now we need to start adding integration data to payers so I stopped from creating the contents yet.